### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10361,6 +10381,9 @@ detinations->destinations
 detremental->detrimental
 detremining->determining
 detrmine->determine
+detrmined->determined
+detrmines->determinse
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:10385]: Suggested word `detrmines->determinse` is a misspell word.
```text
detrmines->determines
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -30305,6 +30388,8 @@ starups->startups
 statamenet->statement
 statamenets->statements
 stategies->strategies
+stategize->strategize
+stategized->strategized
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:30392]: `ise` version as well?
```text
stategise->strategise
stategised->strategised
```

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [323c5f5](https://github.com/codespell-project/codespell/pull/2418/commits/323c5f5af46edb2f10393ca2f0ebc85f29989c6a)

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:30395]: [323c5f5](https://github.com/codespell-project/codespell/pull/2418/commits/323c5f5af46edb2f10393ca2f0ebc85f29989c6a)

**Pull request merged**